src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/request/AddAddressRequest.kt

```diff
@@ -50,7 +50,7 @@ data class AddAddressRequest(
   @field:Schema(example = "MANUAL", description = "Source of the address", required = true)
   val source: AddressSource,
 
-  @Schema(
+  @field:Schema(
```

Comment:
why the @field?

Reply:
it should be param really - I was following the convention of the rest of the class